tools/profile_endpoints.py

```diff
@@ -274,7 +278,44 @@ def profile_get_concepts(token):
 
 # Main function
 if __name__ == "__main__":
-    authenticate()
+
+    # Create the parser
+    parser = argparse.ArgumentParser(
+        description="Profile endpoints for a specific host and dataset"
+    )
+
+    # Add the arguments
+    parser.add_argument(
+        "--host",
+        type=str,
+        choices=HOSTS.keys(),
+        default="local",
+        help="The host to profile",
+    )
+
+    parser.add_argument(
+        "--dataset",
+        type=str,
+        choices=DATASETS.keys(),
+        default="gcp",
+        help="The dataset to profile",
+    )
+
+    parser.add_argument(
+        "--authenticate",
+        type=bool,
+        default=False,
+        help="If false, use current gcloud authentication. If true, authenticate with "
```

Comment:
can you replace `true` with `True` and `false` with `False`, to match Python types